ci-operator/step-registry/upi/vsphere/windows/pre/upi-vsphere-windows-pre-commands.sh

```diff
@@ -37,6 +49,9 @@ vm_template_username=$(<"${SHARED_DIR}"/windows_vm_template_username.txt)
 # set VM folder
 vm_folder="/${vsphere_datacenter}/vm/${cluster_name}"
```

Comment:
Given no cluster is being created, it may be misleading to lean on `${cluster_name}`?

Reply:
Considering this will mainly be used when a cluster is created I'd like to just leave it as is